src/bin/scribble.ts

```diff
@@ -891,44 +887,21 @@ function pickVersion(versionUsedMap: Map<string, string>): string {
             }
         }
 
-        /**
-         * Merge the CHAs and file maps computed for each target
-         */
-        const groups: SourceUnit[][] = targets.map(
-            (target) => groupsMap.get(target) as SourceUnit[]
-        );
-
-        const [mergedUnits, mergedCtx] = merge(groups);
-
         // Check that merging produced sane ASTs
-        for (const mergedUnit of mergedUnits) {
-            assert(
-                isSane(mergedUnit, mergedCtx),
-                `Merged unit ${mergedUnit.absolutePath} is insane`
-            );
-        }
-
-        for (const target of targets) {
-            const units = groupsMap.get(target) as SourceUnit[];
-            const files = filesMap.get(target) as Map<string, string>;
+        for (const unit of units) {
+            assert(isSane(unit, astCtx), `Merged unit ${unit.absolutePath} is insane`);
```

Comment:
I don't think we need this assert anymore

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/164/commits/42b350e912bd4eef2f59dc60d3bdfcb6f27e67d3